tests/bulkwritecommand/bulkwritecommand-insertOne_error-001.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+MongoDB\Driver\BulkWriteCommand::insertOne() with invalid insert document
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/basic.inc';
+
+$bulk = new MongoDB\Driver\BulkWriteCommand;
+
+// @TODO: FAILED: Expected MongoDB\Driver\Exception\InvalidArgumentException, but no exception thrown!
+echo throws(function() use ($bulk) {
+    $bulk->insertOne(NS, ['' => 1]);
+}, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n\n";
+
+// @TODO: FAILED: Expected MongoDB\Driver\Exception\InvalidArgumentException, but no exception thrown!
+echo throws(function() use ($bulk) {
+    $bulk->insertOne(NS, ["\xc3\x28" => 1]);
+}, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
```

Comment:
This is not permitted with collection bulk write: https://github.com/mongodb/mongo-php-driver/blob/d3a079fb4d6f9173a0c81c0d0250d1304e0292a3/tests/bulk/bulkwrite-insert_error-001.phpt#L10-L27

Reply:
Interesting, I'll have to take another look.

Reply:
This is something that absolutely must be corrected. Currently, you can insert an invalid BSON.

```php
$bulk = new MongoDB\Driver\BulkWriteCommand;
$bulk->insertOne(NS, ["\xc3\x28" => 1]);
$manager->executeBulkWriteCommand($bulk);
$manager->executeQuery(NS, new MongoDB\Driver\Query([]))->toArray();
```

```
Fatal error: Uncaught MongoDB\Driver\Exception\UnexpectedValueException: Detected corrupt BSON data for field path '_id'
```

Reply:
I'll remove this tests as the validation is not required by the spec, as mentioned in https://github.com/mongodb/mongo-c-driver/pull/1590

> Since UTF-8 validation is already broken, there are requests to disable, validation negatively impacts performance, and it is not a spec requirement, this PR excludes default validation in MongoClient.bulkWrite.